src/routes/+layout.svelte

```diff
@@ -81,7 +81,9 @@
 			<slot />
 		</div>
 		<Footer />
-		<Toast />
+		{#if !$inSendingMenu}
```

Comment:
The send component is a <dialog> so it is separated from the rest of the App. Hence I had to add a <Toast /> inside the dialog to be displayed. In order to avoid having two overlapping <Toast />, I display this one only when the inSendingMenu is false